### PR TITLE
Update console.mdx

### PR DESCRIPTION
**Pull request by jdgemm**
included new integration api info

### CODE REVIEW

File: docs/api/console.mdx [hunk 1]
```diff
@@ -778,6 +778,219 @@ _Body Parameters_
...
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | Your API key |
+| content-type (required) | _string_ | application/json          |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required) | _string_ | A unique human-friendly name          |
+| type (required) | _string_ | “azure_hub”             |
```
Comment (vicmgs) [inline docs/api/console.mdx:966]: should be aws type

File: docs/api/console.mdx [hunk 2]
```diff
@@ -778,6 +778,219 @@ _Body Parameters_
...
```
Comment (vicmgs) [inline docs/api/console.mdx:914]: just "azure"

File: docs/api/console.mdx [hunk 3]
```diff
@@ -778,6 +778,219 @@ _Body Parameters_
...
+| azure_hub_policy_name (required) | _string_ | Azure IoT Hub policy name            |
```
Comment (vicmgs) [inline docs/api/console.mdx:915]: azure_policy_name

File: docs/api/console.mdx [hunk 4]
```diff
@@ -778,6 +778,219 @@ _Body Parameters_
...
+| azure_hub_name (required) (required)    | _string_ | Azure IoT Hub name |
+| azure_hub_primary_key (required) | _string_ | Azure IoT Hub primary key |
```
Comment (vicmgs) [inline docs/api/console.mdx:917]: azure_primary_key

File: docs/api/console.mdx [hunk 5]
```diff
@@ -778,6 +778,219 @@ _Body Parameters_
...
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required) | _string_ | A unique human-friendly name           |
+| type (required) | _string_ | “http”             |
+| endpoint (required) | _string_ | endpoint             |
+| method (required)    | _string_ | HTTP method |
+| headers (optional) | _string_ | Map of header keys and values { “content-type”: “application/json” } |
```
Comment (vicmgs) [inline docs/api/console.mdx:810]: not "_string_" but a "map"

File: docs/api/console.mdx [hunk 6]
```diff
@@ -778,6 +778,219 @@ _Body Parameters_
...
+
```
Comment (vicmgs) [inline docs/api/console.mdx:811]: add row for 

url_params (optional) | map | Map for payload interpolation { "id": "{{device.id}}"}

**Review by kent-williams — APPROVED** [review]

**Pull request merged**